Formula/rust.rb

```diff
@@ -33,6 +39,7 @@ class Rust < Formula
   end
 
   option "with-llvm", "Build with brewed LLVM. By default, Rust's LLVM will be used."
+  option "with-racer", "Builds racer code completion tool, and retains rust sources."
```

Comment:
I'd suggest making this `without-default` for now so that CI actually tests it, regardless of whether the final decision is that we make it non-default.